src/ahbichtfunctionsclient/__init__.py

```diff
@@ -37,3 +43,46 @@ async def get_condition_expression(self, package_key: str) -> PackageKeyConditio
                 response_body = await response.json()
                 result = PackageKeyConditionExpressionMappingSchema().load(response_body)
                 return result
+
+
+# pylint: disable=too-few-public-methods
+class HochfrequenzConditionResolver:
+    """
+    A package resolver that uses a REST API (by Hochfrequenz) to request a condition text for a given condition key.
+    Note that this resolver requires an internet connection to work and the Hochfrequenz API to be up and running.
+    Consider using this resolver to retrieve package information once and then dump them into something fast and stable
+    like e.g. a JSON file, a database or feed its results into a hardcoded package resolver once on startup.
+    Relying on external web services is prone to be a bottleneck for your application.
```

Comment:
ja, das gilt für beide (package und condition) resolver gleichermaßen. _Vielleicht_ könnte man das auch in den gemeinsamen Module docstring packen.
Ansonsten ist das auch in Ordnung zu doppeln.

Reply:
271af29b203607bd7ca6ffc2958bb519b61651a0